src/test/java/be/seeseemelk/mockbukkit/ServerMockTest.java

```diff
@@ -1391,6 +1416,33 @@ void testSetMotd()
 		assertEquals("Test", server.getMotd());
 	}
 
+	@Test
+	void getRegistry() {
```

Comment:
This should be split up into two tests, one for implemented types and one for non-implemented types. A parameterized test can also be used to reduce duplication.